Controller/ContentTypeController.php

```diff
@@ -350,4 +372,23 @@ public function deleteContentTypeAction(Request $request, $contentTypeId)
 
         return $this->redirectToRouteAfterFormPost('admin_contenttypeView', ['contentTypeId' => $contentTypeId]);
     }
+
+    /**
+     * Return the highest prioritized language that exists in $languageCodes, or if none, return $fallbackLanguageCode.
+     *
+     * @param string[] $languageCodes Language codes
+     * @param string $fallbackLanguageCode Fallback language code
+     *
+     * @return string Language code
+     */
+    private function getPrioritizedLanguage($languageCodes, $fallbackLanguageCode)
```

Comment:
Yeah, the usage has changed, so this is possible now (though must be ContentType, not ContentTypeDraft, since it must support both).